pkgs/development/python-modules/pylatex/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, matplotlib
+, numpy
+, ordered-set
+, quantities
+}:
+
+buildPythonPackage rec {
+  pname = "pylatex";
+  version = "1.3.4";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "JelteF";
+    repo = "PyLaTeX";
+    rev = "v${version}";
+    hash = "sha256-7JE8xKryf7sjvlUjDH7jUMm7CyirjPbmWo+reWeGIlc=";
+  };
+
+  passthru.optional-dependencies = {
+    withMatplotlib = [ matplotlib ];
+    withMatrices = [ numpy ];
+    withQuantities = [ numpy quantities ];
+  };
+
+  propagatedBuildInputs = [ ordered-set ] ++
+    passthru.optional-dependencies.withMatplotlib ++
+    passthru.optional-dependencies.withMatrices ++
+    passthru.optional-dependencies.withQuantities;
```

Comment:
Should these be part of the default runtime closure, or are they only needed at check time? If the latter, consider moving them to `checkInputs`

Reply:
They should be present at runtime.
A lot of features depend on those libraries. For example to translate matrices from numpy into LaTex.

Reply:
The please add a comment stating that you've deemed their inclusion common enough to be the default.

Reply:
I did :)